### PR TITLE
New Swap Widget

### PR DESCRIPTION
**Pull request by ted-palmer**

### CODE REVIEW

File: packages/ui/src/components/widgets/WidgetContainer.tsx [hunk 1]
```diff
@@ -69,65 +69,50 @@ const WidgetContainer: FC<WidgetContainerProps> = ({
...
+          setAddressModalOpen(open)
+        }}
+        onConfirmed={(address) => {
+          setCustomToAddress(address)
+        }}
+        onClear={() => {
+          setCustomToAddress(undefined)
+        }}
+      />
+      {/* <WidgetFooter /> */}
```
Comment (pedromcunha) [inline packages/ui/src/components/widgets/WidgetContainer.tsx:115]: Did we loose the widget footer? We should probably remove this code

File: packages/ui/src/components/widgets/WidgetContainer.tsx [hunk 2]
```diff
@@ -69,65 +69,50 @@ const WidgetContainer: FC<WidgetContainerProps> = ({
...
-        direction="column"
-        css={{
-          width: '100%',
-          borderRadius: 'widget-border-radius',
-          overflow: 'hidden',
-          backgroundColor: 'widget-background',
-          boxShadow: 'widget-box-shadow',
-          border: 'widget-border',
-          p: '4',
-          minWidth: 300,
```
Comment (pedromcunha) [inline packages/ui/src/components/widgets/WidgetContainer.tsx:82]: Did we lose the midWidth and maxWidth?

**Review by pedromcunha — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/widgets/WidgetContainer.tsx (hunk 1 above)
Comment (ted-palmer) [reply packages/ui/src/components/widgets/WidgetContainer.tsx:115]: Yep we did. Will remove the comment 👍

File: packages/ui/src/components/widgets/WidgetContainer.tsx (hunk 2 above)
Comment (ted-palmer) [reply packages/ui/src/components/widgets/WidgetContainer.tsx:82]: The minWidth and maxWidth container just got moved to the SwapWidget and ChainWidgets

**Pull request merged**